planner/core/testdata/integration_suite_out.json

```diff
@@ -1964,21 +1964,20 @@
       {
         "SQL": "select * from t where a > 1 order by f",
         "Plan": [
-          "IndexLookUp_14 3333.33 136747.00 root  ",
```

Comment:
Expected, model2 prefers to use single-read Scan instead of double-read Lookup if not much data to access.